yaksh/python_assertion_evaluator.py

```diff
@@ -49,6 +49,8 @@ def check_code(self, user_answer, file_paths, test_case):
             err = "{0} {1} in: {2}".format(type.__name__, str(value), text)
         except TimeoutException:
             raise
+        except Exception:
```

Comment:
Yes. I have removed the Timeout Exception but I decided to catch and raise all other exceptions (except AssertionError).